profiles/arch/ia64/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Thomas Bettler <thomas.bettler@gmail.com> (2021-12-24)
+# dev-python/rdflib isn't keyworded here. Waiting for bug #814179
+media-libs/lv2 doc
```

Comment:
Yep!

Reply:
Hm, it's unconditionally RDEPENDing on it too... https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/lv2/lv2-1.18.2.ebuild#n38

Is that correct?

Reply:
Hmm, thank you for spotting this :) We need to take another solution...